package/common/cpp/react/renderer/components/RNCSlider/RNCSliderState.cpp

```diff
@@ -0,0 +1,9 @@
+#include "RNCSliderState.h"
+
+namespace facebook {
```

Comment:
We don't need that file if it's empty.
As long as the header is included anywhere with all the definitions then we should be fine while compilation. We only define source file with the definitions in it, if we want to compile the code once and link it anywhere when linking while building.